packages/core/lottie-react-native.podspec

```diff
@@ -2,8 +2,6 @@ require 'json'
 
 package = JSON.parse(File.read(File.join(__dir__, "package.json")))
 
-folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
```

Comment:
Any reason on why this is removed? It's being reference later on line34 in the new change.

Reply:
Maybe this? https://github.com/facebook/react-native/blob/90fb73e83caf087659b6b3173edd457b4f1d1987/packages/react-native/scripts/cocoapods/new_architecture.rb#L11C7-L11C19

Reply:
this can be removed, in theory, because it is already declared in the function `install_modules_dependencies` so you don't need to specify it... but I overlooked line 34.. so we need to restore it. sorry. :(

Reply:
No worries! I guess that might describe why `dynamic` is not working?

Reply:
Good! When you find some time, let's revert this for backward compatibility 😉